app/src/main/res/layout/view_show_raw_metadata_dialog.xml

```diff
@@ -221,6 +223,15 @@
                 app:layout_constraintTop_toBottomOf="@id/unique_title"
                 tools:text="Some Example value" />
 
+            <View
+                android:id="@+id/bottomPadding"
+                android:layout_width="10dp"
+                android:layout_height="40dp"
+                android:background="@color/text_red"
```

Comment:
I get a red box in the metadata view now that shouldn't be there. I also think that the "padding" should be less than the amount used here.

Reply:
aaaa my mistake that was for testing only. will remove in a second

Reply:
the bottom padding size here is to allow for easier dismissal of the dialog on some screen sizes. removed the red background as that was a mistake on my end.